pytorch_to_returnn/import_wrapper/torch_wrappers/function.py

```diff
@@ -0,0 +1,17 @@
+from .module import WrappedModuleBase
+
+
+class Functional(WrappedModuleBase):
```

Comment:
I wonder if we should rename this to describe it a bit better, e.g. to `WrappedTorchFunction` or so

Reply:
Yes, maybe.